src/message/messageActionSheet.js

```diff
@@ -148,6 +148,8 @@ const isSentBySelfAndNarrowed = ({ message, auth, narrow }: AuthMessageAndNarrow
 const isSentBySelf = ({ message, auth }: AuthAndMessageType): boolean =>
   auth.email === message.sender_email;
 
+const isDeleted = ({ message }): boolean => !(message.content === '<p>(deleted)</p>');
```

Comment:
The name should be `isNotDeleted` right?
You can also simplify from `!(a === b)` to `(a !== b)`